web_app/dist/main.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <link rel="icon" type="images/png" href="images/icon.png" />       
+        <meta charset="utf-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <title>TaskViewer</title>
+        <meta name="description" content="Schedule tasks, without the clutter">
+        <link rel="stylesheet" href="main.css">
+        
+    </head>
+ 
+    <body>  
+        
+    <div id="sticky-header">
+        <div class="navbar">
+            <img id="logo" src="images/icon_small.png" alt="TaskViewer Logo">
+            <a>Log out</a>
+            <a>Sort</a>
+            <a>Filter by</a>
+            <input id="search-bar" type="text" placeholder="Search..">
+        </div>
+        <div class="button-class">
+            <button id="create-task-button"class="button">+</button>    
+        </div>
+        <div class="button-class">
+            <button id="delete-task-button" class="button">x</button>    
+        </div>
+    </div>    
+
+    
+    <div id="content">
+    <div id="main">
```

Comment:
Why do we need both `#content` and `#main`?? Also I don't think this is valid HTML, the `#content` div ends up not being closed

If we need both due to how the background work or some reason, let's rename them to be more descriptive, and also fix the tabbing

Reply:
yep, you're right, fixed